etc/kayobe/kolla/globals.yml

```diff
@@ -2,31 +2,3 @@
 ---
 docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
 docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}/gpg"
-
-{% if kolla_base_distro == 'centos' %}
-barbican_tag: wallaby-20220311T133847
-bifrost_tag: wallaby-20220825T103439
```

Comment:
and need Bifrost here for CentOS

Reply:
didn't build seed images, will run new job to do it